### PR TITLE
Playwright workflow for modifying a stream and wrapping tokens

### PR DESCRIPTION
**Pull request by elvijsTDL**

### CODE REVIEW

File: tests/helpers/ethHelper.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  wallet.provider
+);
+const fUSDC = new ethers.Contract(fUSDCAddress, erc20ABI, wallet.provider);
+const CFA = new ethers.Contract(CFAAddress, cfAv1ForwarderABI, wallet.provider);
+
+export async function revokeAllowanceIfNeccesary() {
+  await fUSDC
+    .allowance(
+      "0x7c5de59A1b31e3D00279A825Cb95fAEDb09eA9FD",
+      "0x8aE68021f6170E5a766bE613cEA0d75236ECCa9a"
```
Comment (kasparkallas) [inline tests/helpers/ethHelper.ts:28]: What accounts are these?

File: tests/helpers/abis/wagmi-generated.ts [hunk 2]
```diff
@@ -0,0 +1,1552 @@
+// Generated by @wagmi/cli@1.1.0 on 6/12/2023 at 10:27:05 AM
```
Comment (kasparkallas) [inline tests/helpers/abis/wagmi-generated.ts:1]: Why is this file necessary?

File: tests/specs/widget.spec.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  let widgetPage = new WidgetPage(page);
+  await widgetPage.connectWallet();
+  await widgetPage.selectPaymentNetwork("Goerli");
+  await widgetPage.selectPaymentToken("fUSDCx");
+  await widgetPage.clickContinueButton();
+  await widgetPage.setWrapAmount("0");
+  await widgetPage.clickContinueButton();
+  await widgetPage.validateAndSaveSenderAndReceiverAddresses(
+    "0x7c5de59A1b31e3D00279A825Cb95fAEDb09eA9FD",
+    "0xF26Ce9749f29E61c25d0333bCE2301CB2DFd3a22"
```
Comment (kasparkallas) [inline tests/specs/widget.spec.ts:19]: What accounts are these?

File: .gitignore [hunk 4]
```diff
@@ -197,4 +197,7 @@ fabric.properties
 packages/widget/dist
 
 # Sentry Auth Token
-.sentryclirc
\ No newline at end of file
+.sentryclirc
+
+## Webstorm files
+.idea
```
Comment (kasparkallas) [inline .gitignore:203]: It's not right to ignore the whole .idea folder. The granular .idea ignores are above.

File: tests/helpers/abis/wagmi-generated.ts (hunk 2 above)
Comment (elvijsTDL) [reply tests/helpers/abis/wagmi-generated.ts:1]: Used for web3 actions before/during tests e.g. deleting a flow , revoking allowance in case of a failed test before , checking actual balance on chain and what is visible in the widget etc. 

Could re-use the ABIs from the app, but wanted to keep anything test related separate like we have for other test suites

File: tests/specs/widget.spec.ts (hunk 3 above)
Comment (elvijsTDL) [reply tests/specs/widget.spec.ts:19]: The widget testing wallet and Goerli rebounder

File: tests/helpers/ethHelper.ts (hunk 1 above)
Comment (elvijsTDL) [reply tests/helpers/ethHelper.ts:28]: The widget testing wallet and fUSDCx contract

File: .gitignore (hunk 4 above)
Comment (elvijsTDL) [reply .gitignore:203]: I've always removed the whole folder to not mess up others settings, but sure, ill make it more granular for the files that are left in my workspace

**Review by kasparkallas — APPROVED** [review]
